cfg/sorted_vector_helpers.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SORBET_SORTED_VECTOR_HELPERS_H
+#define SORBET_SORTED_VECTOR_HELPERS_H
+
+#include <vector>
+
+namespace sorbet::cfg {
+// Given sorted vectors `data` and `toRemove` (with no duplicate entries), removes items in `toRemove` from `data`.
+void setDifferenceInplace(std::vector<int> &data, const std::vector<int> &toRemove);
+
+// Merges sorted vectors `info` and `from` (which contain no duplicate entries) without introducing duplicates (set
+// union).
+void setUnionInplace(std::vector<int> &info, const std::vector<int> &from);
+} // namespace sorbet::cfg
```

Comment:
Would you mind making these static functions on a class? That's the style we've been using elsewhere in the codebase, and I'd like to keep it consistent.

Reply:
Sure.

Reply:
Done.